src/natcap/invest/coastal_vulnerability.py

```diff
@@ -2985,10 +3078,14 @@ def _validate_habitat_table_paths(habitat_table_path):
         try:
             pygeoprocessing.get_vector_info(base_habitat_path)
         except ValueError:
-            bad_paths.append(base_habitat_path)
+            try:
+                pygeoprocessing.get_raster_info(base_habitat_path)
+            except ValueError:
+                bad_paths.append(base_habitat_path)

```

Comment:
This might be another use case for `pygeoprocessing.get_gis_type`, since `get_raster_info` and `get_vector_info` will both return a bunch of other information that aren't strictly needed.